integration-tests/15_achievements.ts

```diff
@@ -575,532 +575,478 @@ function generateLectures(dates: Date[], match: Match, user: User) {
+        achievementTemplateCreate(data: $template)
+    }`,
+        { template }
+    );
+
+    if (!skipActivation) {
+        await adminClient.request(`mutation ActivateTemplate {
+            achievementTemplateActivate(achievementTemplateId: ${result.achievementTemplateCreate})
+        }`);
+    }
+}
+
 const createStudentOnboardingTemplates = async () => {
-    await prisma.achievement_template.create({
-        data: {
-            name: 'Onboarding abschließen',
-            templateFor: achievement_template_for_enum.Global,
-            group: 'student_onboarding',
-            groupOrder: 1,
-            stepName: 'Verifizieren',
-            type: achievement_type_enum.SEQUENTIAL,
-            subtitle: 'Jetzt durchstarten',
-            description: 'Dieser Text muss noch geliefert werden.',
-            image: 'Puzzle_00',
-            achievedImage: '',
-            actionName: 'E-Mail erneut senden',
-            actionRedirectLink: '',
-            actionType: achievement_action_type_enum.Action,
-            condition: 'student_verified_events > 0',
-            conditionDataAggregations: { student_verified_events: { metric: 'student_onboarding_verified', aggregator: 'count' } },
-            isActive: true,
-        },
+    await createTemplate({
+        name: 'Onboarding abschließen',
+        templateFor: 'Global',
+        group: 'student_onboarding',
+        groupOrder: 1,
+        stepName: 'Verifizieren',
+        type: 'SEQUENTIAL',
+        subtitle: 'Jetzt durchstarten',
+        description: 'Dieser Text muss noch geliefert werden.',
+        image: 'Puzzle_00',
+        achievedImage: '',
+        actionName: 'E-Mail erneut senden',
+        actionRedirectLink: '/somewhere',
+        actionType: 'Action',
+        condition: 'student_verified_events > 0',
+        conditionDataAggregations: { student_verified_events: { metric: 'student_onboarding_verified', aggregator: 'count' } },
     });
-    await prisma.achievement_template.create({
-        data: {
-            name: 'Onboarding abschließen',
-            templateFor: achievement_template_for_enum.Global,
-            group: 'student_onboarding',
-            groupOrder: 2,
-            stepName: 'Kennenlerngespräch buchen',
-            type: achievement_type_enum.SEQUENTIAL,
-            subtitle: 'Jetzt durchstarten',
-            description: 'Dieser Text muss noch geliefert werden.',
-            image: 'Puzzle_01',
-            achievedImage: '',
-            actionName: 'Termin vereinbaren',
-            actionRedirectLink: 'https://calendly.com',
-            actionType: achievement_action_type_enum.Action,
-            condition: 'student_appointment_booked_events > 0',
-            conditionDataAggregations: {
-                student_appointment_booked_events: { metric: 'student_onboarding_appointment_booked', aggregator: 'count' },
-            },
-            isActive: false,
```

Comment:
Why was this deactivated? If it stays deactivated the groupOrder is not sequential? Don't we assume this? Or does it only have to be monotonically increasing?